ingestor/chalicelib/gtfs/ingest.py

```diff
@@ -112,6 +123,7 @@ def ingest_feeds(
     force_rebuild_feeds: bool = False,
 ):
     for feed in feeds:
+        feed.use_compact_only()
```

Comment:
Is this okay for all the s3 uploads? This is what mbta-performance will use every half hour